dataactvalidator/interfaces/validatorErrorInterface.py

```diff
@@ -26,7 +26,7 @@ def createFile(self, jobId, filename):
         self.session.commit()
         return fileRec
 
-    def createFileIfNeeded(self, jobId, filename):
+    def createFileIfNeeded(self, jobId, filename = ""):
```

Comment:
Since we're trying to handle the case of no applicable filename, does it make sense to default this parameter to `None` instead of a blank string (to ensure that it inserts to the database as `NULL`)?